beanstalk-deploy.js

```diff
@@ -368,8 +368,8 @@ function main() {
         if (versionAlreadyExists) {
 
             if (!environmentName) {
-                console.error(`You have no environment set, so we are trying to only create version ${versionLabel}, but it already exists in Beanstalk!`);
-                process.exit(2);
+                console.log(`You have no environment set, so we are trying to only create version ${versionLabel}, but it already exists in Beanstalk!`);
```

Comment:
LGTM overall!

Now that I think on it, let's rephrase slightly to `No environment set, but the version ${versionLabel} was found - exiting successfully with no change`

Reply:
Ah yeah this makes sense